Feature/syal 829

ThanhD-AVISPL opened the pull request:
Pull request for VIA Connect Pro.

ThanhD-AVISPL committed:
SYAL-829

Implement monitoring properties.

ThanhD-AVISPL committed:
feature/SYAL-829

ThanhD-AVISPL committed:
feature/SYAL-829

ThanhD-AVISPL committed:
feature/SYAL-829

Add code to support send telnet command to device.

ThanhD-AVISPL committed:
feature/SYAL-829

Delete unused properties, refactor code.

ThanhD-AVISPL committed:
SYAL-830

Implement controlling properties.

ThanhD-AVISPL committed:
feature/SYAL-830

Refactor code and create enum class for DisplayStatusMode.

ThanhD-AVISPL committed:
feature/SYAL-831

Add UT for VIA Connect Pro.

ThanhD-AVISPL committed:
feature/SYAL-831

Update VIAConnectProCommunicatorTest file.

ThanhD-AVISPL committed:
feature/SYAL-830

Add VIAConnectProErrorMetric class, update VIAConnectProCommunicator class.

ThanhD-AVISPL committed:
Update VIAConnectProConstant class

ThanhD-AVISPL committed:
feature/SYAL-830

Update code based on comments.

ThanhD-AVISPL committed:
feature/SYAL-830

Fix code after QA test, refactor the code.

ThanhD-AVISPL committed:
feature/SYAL-830

Remove unused code.

MaksimR-AVISPL commented on pom.xml:
```diff
@@ -0,0 +1,240 @@
+            <artifactId>symphony-api-dal</artifactId>
+            <version>${symphonyApiVersion}</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/symphony-api-dal-5.4.0.jar</systemPath>
```
can you please update SY dependencies to 5.5 since it was released?

ThanhD-AVISPL replied:
Sure, I will update it to 5.5

MaksimR-AVISPL commented on src/main/java/com/avispl/symphony/dal/avdevices/wireless/presentation/kramer/viaconnectpro/VIAConnectProCommunicator.java:
```diff
@@ -0,0 +1,1432 @@
+	 */
+	@Override
+	protected void internalDestroy() {
+		cachedLocalExtendedStatistics.setStatistics(new HashMap<>());
```
would it make sense to call getStatistics().clear() and .getControllableProperties().clear() here instead?

ThanhD-AVISPL replied:
Yes, I will change it as your comment.

ThanhD-AVISPL committed:
Resolve and update code

Update symphony api version to 5.5.0, update internalDestroy method based on comments.
